web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -147,65 +191,80 @@ describe("<SidebarCardsList />", () => {
         renderComponent({
           fundings: [],
           certifications: [],
-          displayCertifications: false,
-          displayFundings: true,
         });
 
         expect(screen.queryByTestId("empty-for-you-message")).not.toBeInTheDocument();
       });
     });
 
     describe("Complete Required Tasks Message", () => {
-      it("displays when there are no top or bottom cards and display Fundings and Certifications are false", () => {
+      it("displays when there are no sideBar cards and display Fundings and Certifications are false", () => {
+        useMockBusiness({
+          profileData: {
+            ...generateProfileData({
+              operatingPhase: operatingPhaseWithDisplayFundingAndDisplayCertificationAreFalse?.id || "",
+            }),
+          },
+        });
+
         renderComponent({});
         const emptyMessage = screen.getByTestId("empty-for-you-message");
         const sanitizedString = removeMarkdownFormatting(Config.dashboardDefaults.completeRequiredTasksText);
         expect(emptyMessage).toHaveTextContent(sanitizedString);
       });
 
       it("does not display when displayFundings is true", () => {
-        renderComponent({ displayFundings: true });
+        useMockBusiness({
+          profileData: {
+            ...generateProfileData({
+              operatingPhase: operatingPhaseWithDisplayFundingIsTrue?.id || "",
+            }),
+          },
+        });
+        renderComponent({});
         expect(screen.queryByTestId("empty-for-you-message")).not.toBeInTheDocument();
       });
 
       it("does not display when displayCertifications is true", () => {
-        renderComponent({ displayCertifications: true });
+        useMockBusiness({
+          profileData: {
+            ...generateProfileData({
+              operatingPhase: operatingPhaseWithDisplayCertificationIsTrue?.id || "",
+            }),
+          },
+        });
+        renderComponent({});
         expect(screen.queryByTestId("empty-for-you-message")).not.toBeInTheDocument();
       });
 
       it("does not display when there are top cards", () => {
-        const topCards = [
+        const sideBarCards = [
           generateSidebarCardContent({
             id: "some-fake-top-card",
-            section: "above-opportunities",
           }),
         ];
-        renderComponent({ topCards });
+        renderComponent({ sideBarCards });
         expect(screen.queryByTestId("empty-for-you-message")).not.toBeInTheDocument();
       });
 
       it("does not display when there are bottom cards", () => {
-        const bottomCards = [
+        const sideBarCards = [
           generateSidebarCardContent({
             id: "some-fake-bottom-card",
-            section: "below-opportunities",
           }),
         ];
-        renderComponent({ bottomCards });
+        renderComponent({ sideBarCards });
         expect(screen.queryByTestId("empty-for-you-message")).not.toBeInTheDocument();
       });
     });
```

Comment:
[sand] Aren't these 2 the same test?

Reply:
Yes, I'll delete the duplicate and update the test name